packages/amplify-gen2-codegen/src/backend/synthesizer.ts

```diff
@@ -201,10 +201,12 @@ export class BackendSynthesizer {
     userPoolClientAttributesMap.set('ExplicitAuthFlows', 'authFlows');
     userPoolClientAttributesMap.set('AllowedOAuthFlows', 'flows');
 
-    const test = factory.createCallExpression(
-      factory.createPropertyAccessExpression(factory.createIdentifier('userPool'), factory.createIdentifier('addClient')),
-      undefined,
-      [factory.createStringLiteral('NativeAppClient'), this.createNestedObjectExpression(userPoolClient, userPoolClientAttributesMap)],
+    const nativeUserPoolClientExpressionStatement = factory.createExpressionStatement(
```

Comment:
there was a semicolon missing after `addClient` expression. Wrapping in a statement adds it.